src/main/java/uk/gov/hmcts/reform/lrdapi/service/LrdBuildingLocationService.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.lrdapi.service;
+
+import uk.gov.hmcts.reform.lrdapi.controllers.response.LrdBuildingLocationResponse;
+
+import java.util.List;
+
+public interface LrdBuildingLocationService {
+
```

Comment:
trivial comment can we please refactor and rename to IlrdBuildingLocationService

Reply:
Just followed the existing naming conventions in the LRD API. But sure, I will change this.